TriangleNet_Engine/Compute/DelaunayTriangulation.cs

```diff
@@ -136,6 +139,93 @@ public static List<Polyline> DelaunayTriangulation(this Polyline outerCurve, Lis
 
             return meshPolylines;
         }
+
+        /***************************************************/
+
+        [Description("Creates a list of closed Delaunay polyline cells from a list of coplanar points. Each cell will correspond to a face in a Delaunay mesh.")]
+        [Input("points", "The points to connect to Delanuay cells. All points need to be coplanar ")]
+        [Input("plane", "Optional plane of the points. Needs to be coplanar with all of the points. If no plane is provided, a best fit plane through the points will be used.")]
+        [Input("tolerance", "Numeric tolerance to be used by the method.", typeof(Length))]
+        [Output("curves", "List of closed polyline representing faces in a Delaunay.")]
+        public static List<Polyline> DelaunayTriangulation(List<Point> points, Plane plane = null, double tolerance = Tolerance.Distance)
+        {
+            //Preform check all inputs that triangulation can be done
+            if (points == null || points.Count < 3)
+            {
+                Reflection.Compute.RecordError("Insuffient points for triangulation. Please provide at least 3 Points!");
```

Comment:
Sp. `Insufficient` rather than `Insuffient`

Reply:
Is the `!` consistent with other errors in the past? Seems a little 'OTT' to 'shout' at the users?

Reply:
Ha! Yep - I might remove the use of explanation marks 😄

Reply:
but... what if I want to shout? 😢

Reply:
Then you have to do so quietly :wink: